### DISCUSSION TITLE
SyntaxError: Unexpected token '<', "<!doctype "... is not valid JSON

### DISCUSSION THREAD

**Discussion by web1989** [GrapesJS/grapesjs#5292] [Q&A]
I'm getting this error in the console, I've already searched and I couldn't solve it.
 
<img width="482" alt="JsonError" src="https://github.com/GrapesJS/grapesjs/assets/5933615/c3bf40b8-be5b-48b4-9aa7-161a4f33ba6b">

My codes

```
<script type="text/javascript">
       window.onload = () => {
        window.editor = grapesjs.init({
          height: '100%',
          container: '#gjs', 
          fromElement: true,
          clearOnRender: true,
          plugins: [ 
              'gjs-blocks-basic',
              'grapesjs-preset-webpage',
              ],
          pluginsOpts: {
            'grapesjs-preset-webpage': {},
             'gjs-blocks-basic': { flexGrid: true },
          },
        storageManager: {
           type: 'remote',
            contentTypeJson: true,
            autosave: false,
            autoload: true,
            params: {},
            storeHtml: true,
            storeCss: true,
            urlStore: '{{ route('update', $page_id) }}',
            urlLoad: '{{ route('show', $page_id) }}'
          }
 
        });
        
       };
    </script>
```

localhost/page/1/get output:
`{"gjs-html":"[]","gjs-css":"[]"}`

Laravel controller
```
public function get(Request $request, $id)
{
       $page = Page::findOrFail($id);
       $data = [
            'gjs-html' => $page->html,
            'gjs-css' => $page->css,
        ];
       return response()->json($data);
}
```
I'm using Laravel 8, Could anyone tell me a solution?

**Comment by artf** [comment]
You have to debug your endpoint, you're clearly not returning the JSON but an HTML document.

You're also using an old configuration for the storageManager + trying to rely on HTML/CSS for project editing which is something [we strongly discourage](https://grapesjs.com/docs/modules/Storage.html#project-data).
Please [read carefully the documentation](https://grapesjs.com/docs/modules/Storage.html).
